features/step_definitions/error_page_steps.rb

```diff
@@ -0,0 +1,15 @@
+Given(/^that I visit the internal server error page in my "([^"]*)"$/) do |language|
+  locale = language_to_locale(language)
+
+  internal_server_error_page.load(locale: locale)
+end
+
+Given(/^that I visit a non\-existent page in my "([^"]*)"$/) do |language|
+  locale = language_to_locale(language)
+
+  page_not_found_page.load(locale: locale)
```

Comment:
If this is the step to visit a non-existent page why is visiting the "404" page which exists on our app? Why not trying to access another like "/this-page-does-not-exist" url?

Reply:
@tomas-stefano there's a spec for it already. https://github.com/moneyadviceservice/frontend/blob/master/spec/requests/404_spec.rb#L12

This test is to examine the message we get back from the rendered 404/500 page itself.

Reply:
But what user goes direct to the 404 page? Shall we not mimic the user behaviour on our cucumber specs?